src/main/resources/templates/home.html

```diff
@@ -1,10 +1,16 @@
 <!DOCTYPE html>
-<html lang="en">
+<html xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
 <head>
-    <meta charset="UTF-8">
-    <title>Home</title>
+    <div th:replace="fragments/header :: header"></div>
 </head>
 <body>
-<h1>Welcome to this amazing application!</h1>
+<div th:replace="fragments/home-navbar :: home-navbar"/>
+<div class="body">
+<h1>Discover how even small actions can have an impact.</h1>
+    <img class="irc_mi" src="http://www.heart.org/-/media/aha/h4gm/article-images/how-to-eat-more-fruits-and-veg.jpg?h=800&amp;la=en&amp;w=1200&amp;hash=F662753067CDEBD879359EB35869072B4F1CCFA3" alt="Image result for images of vegetables and fruits" onload="typeof google==='object'&amp;&amp;google.aft&amp;&amp;google.aft(this)" width="530" height="353" style="margin-top: 0px;">
```

Comment:
Careful when using someone else images. You can run into copyright issues. And as well if you want to use any image it should be in your project, not linked as static resource from other page.